### PR TITLE
Add Title, Image, and Date to social shares

### PR DESCRIPTION
**Pull request by sdotson**
For #97 . 
screen shot 2017-04-17 at 10 58 01 am
screen shot 2017-04-17 at 10 23 54 am

### CODE REVIEW

File: src/components/SocialBtns/facebookSDK.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+ /* eslint-disable */
```
Comment (pfarnach) [inline src/components/SocialBtns/facebookSDK.js]: Make sure to enable eslint at the end of the file

http://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments

Comment (sdotson) [inline src/components/SocialBtns/facebookSDK.js]: Hey thanks for the review. Latest re-enables eslint at the end of the file

**Comment by sdotson** [comment]
@dorono, good eye! Since the buttons don't work with localhost, I temporarily put in an arbitrary image url (the logo on staging/heroku server) to do the screenshot. The code as it stands now in this PR should be taking the `featuredImageUrl` being used in the EventDetails component and passing it down as a prop to SocialBtns

File: src/components/config.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const facebookAppId = '339880856413925';
```
Comment (pdw207) [inline src/components/config.js]: Maybe move to `src/config.js`

File: src/components/SocialBtns/facebookSDK.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    };
+    (function(d, s, id){
+      var js, fjs = d.getElementsByTagName(s)[0];
+      if (d.getElementById(id)) {return;}
+      js = d.createElement(s); js.id = id;
+      js.src = "//connect.facebook.net/en_US/sdk.js";
+      fjs.parentNode.insertBefore(js, fjs);
+    }(document, 'script', 'facebook-jssdk'));
+  },
+  share: function({ title, eventDate, picture, eventUrl, description }) {
```
Comment (pdw207) [inline src/components/SocialBtns/facebookSDK.js]: Can you use ES6 syntax share({title ...}): {}

File: src/components/SocialBtns/facebookSDK.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+ /* eslint-disable */
```
Comment (pdw207) [inline src/components/SocialBtns/facebookSDK.js]: Maybe move to `src/services`? What do you think. It doesn't feel like its a component.

File: src/components/SocialBtns/SocialBtns.js [hunk 5]
```diff
@@ -1,42 +1,57 @@
...
-      </a>
-    </div>
-  );
-};
+export default class SocialBtns extends Component {
+  componentDidMount() {
+    facebookSDK.init();
+  }
+
+  render({ title, startDate, picture, description } = this.props) {
```
Comment (pdw207) [inline src/components/SocialBtns/SocialBtns.js]: Nice! Why do I never do this.

**Comment by pdw207** [comment]
Great job! Thanks.

File: src/components/config.js (hunk 2 above)
Comment (pdw207) [inline src/components/config.js]: The app id is `274135313007103`

File: src/components/SocialBtns/facebookSDK.js (hunk 3 above)
Comment (sdotson) [inline src/components/SocialBtns/facebookSDK.js]: sure can

File: src/components/config.js (hunk 2 above)
Comment (sdotson) [inline src/components/config.js]: in latest.

**Comment by sdotson** [comment]
I've addressed all comments but noticed something with the new facebook app id: The share dialogue won't work with everybody unless we make the app public. If it's too early for that we can designate a few testers for the app and it'll work for them.

Below are screenshots of what I see when I click facebook share now and where one could change the setting to make the app public in facebook (using a screenshot of my test app as an example).

screen shot 2017-04-20 at 9 41 02 am
screen shot 2017-04-20 at 9 37 50 am

**Comment by pdw207** [comment]
Try it now.

**Comment by sdotson** [comment]
Looks like that error is gone now! @pdw207

**Comment by pdw207** [comment]
Had to whitelist the domain name. Now it works on staging.
screen shot 2017-04-20 at 10 28 58 am

**Pull request merged**